airbyte/cli.py

```
@@ -0,0 +1,341 @@
+
+    if config:
+        print("Running connector check...")
+        connector_obj.set_config(json.loads(config.read_text(encoding="utf-8")))
+        connector_obj.check()
+
+
+@click.command()
+@click.option(
+    "--source",
+    type=str,
+    help="The source name, with an optional version declaration. If a path is provided, the source will be loaded from the local path. If the string `'.'` is provided, the source will be loaded from the current working directory.",
```

Ruff

Line too long (231 > 100)

(E501)